jwql/database/reset_database.py

```diff
@@ -24,23 +24,40 @@
     ``postgresql+psycopg2://user:password@host:port/database``.
 """
 
-from jwql.database.database_interface import base, set_read_permissions
+import argparse
+
+from jwql.database.database_interface import base, set_read_permissions, INSTRUMENT_TABLES
 from jwql.utils.utils import get_config
 
 
 if __name__ == '__main__':
 
+    parser = argparse.ArgumentParser(description='Reset JWQL database tables')
+    parser.add_argument('instrument', metavar='INSTRUMENT', type=str,
+                        help='instrument tables to reset ("all" for all)',
+                        default='all')
+    args = parser.parse_args()
+
+    instrument = args.instrument
+
     connection_string = get_config()['connection_string']
     server_type = connection_string.split('@')[-1][0]
 
     assert server_type != 'p', 'Cannot reset production database!'
 
-    prompt = ('About to reset all tables for database instance {}. Do you '
-              'wish to proceed? (y/n)\n'.format(connection_string))
+    prompt = ('About to reset {} tables for database instance {}. Do you '
+              'wish to proceed? (y/n)\n'.format(instrument, connection_string))
     response = input(prompt)
 
     if response.lower() == 'y':
```

Comment:
Should we add something here for `n` and user input that aren't `y/n`?

```
if response.lower() == 'y':
    delete tables
elif response.lower() == 'n'
    print('selected n, exiting')
else:
    print("{} is not valid input, exiting".format(input)) 
```

Might be overkill, but besides that, this LGTM!